front-end/src/components/filter/DisplayFilters.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+
+const DisplayFilters = ({ removeFilter, type, options }) => {
+    return (
+        <Container>
+            {options?.map((item, i) => (
+                <p key={i} onClick={(() => removeFilter(type, item))}>{item}</p>
```

Comment:
Use descriptive variable names.
```suggestion
                <p key={index} onClick={(() => removeFilter(type, item))}>{item}</p>
```